apps/vr-tests-react-components/src/utilities/TestWrapperDecorator.tsx

```diff
@@ -61,37 +53,23 @@ const useNoAnimationStyles = makeStyles({
     },
   },
 });
-export const TestWrapperDecoratorNoAnimation: DecoratorFunction<ExtendedStoryFnReturnType> = story => {
-  const noAnimationStyles = useNoAnimationStyles();
-  const className = mergeClasses(noAnimationStyles.root, 'testWrapper');
+export const TestWrapperDecoratorPauseAnimation: DecoratorFunction<ExtendedStoryFnReturnType> = story => {
+  const pausedAnimationStyles = usePausedAnimationStyles();
+  const className = mergeClasses(pausedAnimationStyles.paused, 'testWrapper');
   return (
     <div style={{ display: 'flex' }}>
       <div
         className={className}
         style={{
           padding: '10px',
-          overflow: 'hidden',
         }}
       >
         {story()}
       </div>
     </div>
   );
 };
-export const TestWrapperDecoratorPauseAnimation: DecoratorFunction<ExtendedStoryFnReturnType> = story => {
-  const noAnimationStyles = useNoAnimationStyles();
-  const className = mergeClasses(noAnimationStyles.paused, 'testWrapper');
-  return (
-    <div style={{ display: 'grid' }}>
-      <div
-        className={className}
-        style={{
-          padding: '10px',
-          overflow: 'hidden',
-        }}
-      >
-        {story()}
-      </div>
-    </div>
-  );
+export const PauseAnimationDecorator: DecoratorFunction<ExtendedStoryFnReturnType> = story => {
+  const pausedAnimationStyles = usePausedAnimationStyles();
+  return <div className={pausedAnimationStyles.paused}>{story()}</div>;
 };
```

Comment:
It would be best to have _only_ `PauseAnimationDecorator`, and remove `TestWrapperDecoratorPauseAnimation` entirely. It looks like the latter is used by Skeleton only -- could you update Skeleton to use `PauseAnimationDecorator` in this PR as well?